includes/Modules/Sign_In_With_Google.php

```diff
@@ -359,17 +386,25 @@ private function render_signinwithgoogle() {
 			'shape' => $settings['shape'],
 		);
 
-		// Whether buttons will be rendered/transformed on this page.
-		$render_buttons = $is_wp_login || $is_woocommerce_login;
-		$render_one_tap = ! empty( $settings['oneTapEnabled'] ) && ( $is_wp_login || ! is_user_logged_in() );
+		// Whether this is a WordPress/WooCommerce login page.
+		$is_login_page = $is_wp_login || $is_woocommerce_login;
 
-		// If we aren't rendering buttons or One-tap, return early.
-		if ( ! $render_buttons && ! $render_one_tap ) {
-			return;
-		}
```

Comment:
If the user has already logged in and this is not the WordPress login page, then we should return early to not render the script.

Reply:
There are a lot of varied logic paths (which kind of page the user is on and whether they're signed in or not, whether one-tap is enabled, etc.) such that the conditions to "return early" are quite specific: essentially it's "one-tap is not enable and the user is signed in and the user is not on the WP login page".

I don't think it's that useful to return early in that one scenario; rather, all of the rendering below is done conditionally.